source/catalog.html

```diff
@@ -64,16 +105,17 @@
         <h1 class="intro__title intro__title--catalog container">Фотографии пользователей</h1>
       </div>
       <section class="catalog">
-        <p class="catalog__slogan slogan container">
+        <p class="catalog__slogan slogan--catalog container">
           Взгляните на фотографии, которые
           выкладывают пользователи!
           Видите, как не хватает ваших?
         </p>
         <div class="catalog__panorama photo-card photo-card--special">
           <div class="catalog__panorama-img photo-card__img-container">
             <picture>
-              <source media="(min-width: 321px)" srcset="img/catalog/photo-card/panorama-desktop@1x.jpg, img/catalog/photo-card/panorama-desktop@2x.jpg 2x">
-              <img class="photo-card__img photo-card__img--special" src="img/catalog/photo-card/panorama-mobile@1x.jpg" srcset="img/catalog/photo-card/panorama-mobile@2x.jpg 2x" alt="Подвесной мост для пешеходов в скайпарке Сочи.">
+              <source type="img/webp" srcset="img/catalog/photo-card/panorama-mobile@1x.webp, img/catalog/photo-card/panorama-mobile@2x.webp 2x" height ="137">
+              <source media="(min-width: 660px)" srcset="img/catalog/photo-card/panorama-desktop@1x.jpg, img/catalog/photo-card/panorama-desktop@2x.jpg 2x" height="509">
```

Comment:
Что-то я тупанул и упустил момент с тем, что у тебя размер именно на `<source>` не работает, он как бы не существует в разметке и размеры меняются через стили
```suggestion
              <source media="(min-width: 660px)" srcset="img/catalog/photo-card/panorama-desktop@1x.jpg, img/catalog/photo-card/panorama-desktop@2x.jpg 2x">
```